README.md

```diff
@@ -156,6 +164,8 @@ Clone the repository into `idrstream/` with:
 ```sh
```

Comment:
I do not know if `sh` is the right choice. Maybe `console` may be a better choice because it can let the reader know this is done within the `console` 

`sh` assumes it's a shell script, unless it is!

Reply:
Implemented in https://github.com/WayScience/IDR_stream/pull/12/commits/5cf59bb6be7873572cf196ebb2babb3f015f2bb7!